src/coreclr/nativeaot/Runtime/eventtrace_bulktype.cpp

```diff
@@ -312,7 +318,7 @@ int BulkTypeEventLogger::LogSingleType(MethodTable * pEEType)
         // So no other type flags are applicable to set
     }
 
-    ULONGLONG rvaType = osModuleHandle == 0 ? 0 : (ULONGLONG(pEEType) - osModuleHandle);
+    ULONGLONG rvaType = ULONGLONG(pEEType) - osModuleHandle;
```

Comment:
How does this work for dynamically created types?

Reply:
It never did.

If we want to fix that, I think we'll need to build names of these as part of type building just in case they are needed, waste some memory, and send them over as strings.

LogSingleType needs more fixes, it has other problems.

Reply:
ok